packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/BaseClient.java

```diff
@@ -160,7 +163,8 @@ public abstract class BaseClient {
 	private static final ObjectMapper objectMapper = JsonMapper
 		.builder()
 		.findAndAddModules()
-		.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+		.defaultTimeZone(TimeZone.getTimeZone(ZoneOffset.UTC))
+		.withConfigOverride(Instant.class, cfg -> cfg.setFormat(JsonFormat.Value.forPattern("uuuu-MM-dd'T'HH:mm:ss.SSSSX")))
```

Comment:
Is it okey to have only 4 seconds' decimals? I think backend expect to have 6 (microseconds) at least in standard schema.

Reply:
My bad, thought it was 4 digits. I couldn't find this documented anywhere on the website.